javascript/javascript3/week2/Movies/movies_array.js

```diff
@@ -0,0 +1,39 @@
+//1. Create an array of bad movies
+const apiUrl = `https://tinyurl.com/y9zck5xx`;
+// Bad movies rated for <=4
+async function badMoviesArray() {
```

Comment:
here you are prefixing with async. It's only needed if you are using await inside it.